papiea-engine/src/logger.ts

```
@@ -49,16 +42,22 @@ export class WinstonLogger implements Logger {
         if (!this.isValidLevel(logLevel)) {
```

Should we lose the log just because the log level is incorrect ? I suggest we log that we found an invalid log level, and then use error as the default logging here. That way the flow won't break in case of invalid log level.

---

I believe that we shouldn't confuse error logs with incorrect level logs. This way this might bloat the the error log with unexpected logs

---

@Emulebest I agree with @nkntnx here. Logging something is better than silently ignore problems. I rather see bloated logs that point me to the problem which I could then fix, than this opaque `Unsupported logging level ` error which tells me that something somewhere gone wrong. 

If we get an unsupported logging level, I would implicitly make it as a `warning` level, and write that `Unsupported logging level` was used, along with the original log message and potentially its location in the code.

---

Fixed this.